### ISSUE TITLE
App deployment hangs at health check

### ISSUE THREAD

**Issue by frankyhun** [cloudfoundry/cli#795]
### Command
cf push -f manifest.yml

### CLI Version
go-cli 6.16.1+924508c / windows

### Error
CF CLI Tool hangs at GET /v2/apps/xxxx-xxxx-xxxx-xxxx-xxxx/stats

### CF Trace
...
{"0":{"state":"STARTING","since":1457685939.1304915,"debug_ip":null,"debug_port":null,"console_ip":null,"console_port":null}}

REQUEST: [2016-03-11T09:45:49+01:00]
GET /v2/apps/753bf9c1-74e9-463d-996a-ee7282f0634a/stats HTTP/1.1
Host: [deleted]
Accept: application/json
Authorization: [PRIVATE DATA HIDDEN]
Content-Type: application/json
User-Agent: go-cli 6.16.1+924508c / windows

### Platform Details

Windows7 64bit

### Shell
Windows cmd

**Comment by dkoper** [comment]
Hi @frankyhun 

Apologies for the delay.
Can you share with us your app manifest? Does it contain one or multiple apps? Are you behind an HTTP proxy? Does it always hang or only once/intermittently? Does it hang for `cf app` too? When you run `cf curl` with the API mentioned last in the trace, does that hang too?

**Comment by dkoper** [comment]
Closing. Feel free to reopen or submit a new issue if you can reproduce the issue on the latest cf CLI release and you can offer more information for us to investigate.
